code/modules/mob/living/carbon/alien/humanoid/update_icons.dm

```diff
@@ -92,10 +90,10 @@
 	if (head)
 		var/t_state = head.item_state
 		if(!t_state)	t_state = head.icon_state
-		var/image/lying		= image("icon" = 'icons/mob/mob.dmi', "icon_state" = "[t_state]2")
-		var/image/standing	= image("icon" = 'icons/mob/mob.dmi', "icon_state" = "[t_state]")
+		var/image/lying		= image(((head.icon_override) ? head.icon_override : 'icons/mob/head.dmi'), "icon_state" = "[t_state]")
+		var/image/standing	= image(((head.icon_override) ? head.icon_override : 'icons/mob/head.dmi'), "icon_state" = "[t_state]")
```

Comment:
Please just pretend this whole file doesn't exist, it's not actually used anywhere and I only briefly toyed around with making it work before realizing it's unused for a good reason